### DISCUSSION TITLE
Rankup Float/Integer Placeholder Requeriments

### DISCUSSION THREAD

**Discussion by aldoapicella** [Zrips/CMI#4785] [Feature Requests]
Hi! I thought this could be an interesting idea for new ways for players to rankup, for example:

```
<NameHere>Requeriments:
  - %placeholder_1% == 5
  - %placeholder_2% >= %placeholder_3%
```

In this rank, the player will only be able to rankup if %placeholder_1% is equal to 5 and if %placeholder_2% is greater than the output of %placeholder_3%

**Comment by lochlanmcelroy** [comment]
BUMP! This would be really helpful.
